src/Microsoft.Diagnostics.ExtensionCommands/DumpHeapCommand.cs

```diff
@@ -227,6 +230,22 @@ private void ParseArguments()
                 MethodTable = Runtime.Heap.StringType.MethodTable;
             }
 
+            if (!string.IsNullOrWhiteSpace(Generation))
+            {
+                Generation generation = Generation switch
+                {
+                    "gen0" => Diagnostics.Runtime.Generation.Generation0,
+                    "gen1" => Diagnostics.Runtime.Generation.Generation1,
+                    "gen2" => Diagnostics.Runtime.Generation.Generation2,
+                    "loh" => Diagnostics.Runtime.Generation.Large,
```

Comment:
These are good.  Maybe also accept "pinned", "frozen", and "large" to make the command more friendly to inputs?

Reply:
These are used in dumpgen command. I will add friendly names too.